kooplex-proxy/proxy/database.py

```diff
@@ -3,16 +3,24 @@
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 
-POSTGRES_USER = os.getenv('POSTGRES_USER')
-POSTGRES_PASSWORD = os.getenv('POSTGRES_PASSWORD')
-POSTGRES_HOST = os.getenv('POSTGRES_HOST')
-POSTGRES_DB = os.getenv('POSTGRES_DB')
+
+POSTGRES_USER = os.getenv("POSTGRES_USER")
+POSTGRES_PASSWORD = os.getenv("POSTGRES_PASSWORD")
+POSTGRES_HOST = os.getenv("POSTGRES_HOST")
+POSTGRES_PORT = os.getenv("POSTGRES_PORT")
+POSTGRES_DB = os.getenv("POSTGRES_DB")
+POSTGRES_SCHEMA = 'sandbox_public'
+
+SCHEMA_SET = {
+    'sandbox_public',
+    'sandbox_private'
+}
```

Comment:
You should use dictionary instead of set, which will allow you to translate the schema names to the string from Shiny rest call. Of course, this comment is for the next PR.